src/deno_dir.rs

```diff
@@ -420,6 +433,26 @@ fn test_code_fetch() {
   //println!("code_fetch_output {:?}", code_fetch_output);
 }
 
+#[test]
+fn test_code_fetch_no_ext() {
+  let (_temp_dir, deno_dir) = test_setup();
+
+  let cwd = std::env::current_dir().unwrap();
+  let cwd_string = String::from(cwd.to_str().unwrap()) + "/";
+
+  // Assuming cwd is the deno repo root.
+  let module_specifier = "./js/main";
+  let containing_file = cwd_string.as_str();
+  let r = deno_dir.code_fetch(module_specifier, containing_file);
+  assert!(r.is_ok());
+
+  // Assuming cwd is the deno repo root.
+  let module_specifier = "./js/mock_builtin";
+  let containing_file = cwd_string.as_str();
+  let r = deno_dir.code_fetch(module_specifier, containing_file);
+  assert!(r.is_ok());
```

Comment:
Can you check more than just `is_ok()` ? EG
```rust
let code_fetch_output = r.unwrap();
assert_eq!("./js/main.ts", code_fetch_output.module_name);
assert!(code_fetch_output.source_code.len() > 10);
```

Reply:
Added, using `.ends_with` instead

Reply:
Thanks.

Sorry, I should have mentioned this before: can you test `code_fetch_output.filename` too?